scripts/bootstrapper/Dockerfile

```diff
@@ -15,5 +15,7 @@ ARG version
 WORKDIR /build
```

Comment:
If we are no longer using the `copy.sh` file in line 11 above, we should probably just delete the file and reference